testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/example/AbstractCorsExampleAdapterTest.java

```diff
@@ -39,6 +40,7 @@
 /**
  * Created by fkiss.
  */
+@Ignore //Needs a discussion about future work.
```

Comment:
The test is failing because missing domains configuration in /etc/hosts. We need to discuss how to tests this with two possible options: 

1. Move this test to separate module and configure domains localhost-auth and localhost-db in /etc/hosts.
2. Start two app-server containers on different ports and deploy angular-app to first and db-app to second.

Another question is what to do with examples in integration-arquillian testsuite. Whether to preserve them or remove them. 

No, there is no JIRA for this yet.